server/oauth2.go

```diff
@@ -561,3 +567,41 @@ func validateRedirectURI(client storage.Client, redirectURI string) bool {
 	host, _, err := net.SplitHostPort(u.Host)
 	return err == nil && host == "localhost"
 }
+
+// storageKeySet implements the oidc.KeySet interface backed by Dex storage
+type storageKeySet struct {
+	storage.Storage
+}
+
+func (s *storageKeySet) VerifySignature(ctx context.Context, jwt string) (payload []byte, err error) {
```

Comment:
[nit] `ctx` isn't used:
```suggestion
func (s *storageKeySet) VerifySignature(_ context.Context, jwt string) (payload []byte, err error) {
```